astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -217,50 +217,52 @@ def validate(cls, value):
 
     To transform to the Galactocentric frame with the default
     frame attributes, pass the uninstantiated class name to the
-    ``transform_to()`` method of a coordinate frame or
-    `~astropy.coordinates.SkyCoord` object::
+    ``transform_to()`` method of a `~astropy.coordinates.SkyCoord` object::
 
         >>> import astropy.units as u
         >>> import astropy.coordinates as coord
-        >>> c = coord.ICRS(ra=[158.3122, 24.5] * u.degree,
-        ...                dec=[-17.3, 81.52] * u.degree,
-        ...                distance=[11.5, 24.12] * u.kpc)
+        >>> c = coord.SkyCoord(ra=[158.3122, 24.5] * u.degree,
+        ...                    dec=[-17.3, 81.52] * u.degree,
+        ...                    distance=[11.5, 24.12] * u.kpc,
+        ...                    frame=coord.ICRS)
         >>> c.transform_to(coord.Galactocentric) # doctest: +FLOAT_CMP
-        <Galactocentric Coordinate (galcen_coord=<ICRS Coordinate: (ra, dec) in deg
+        <SkyCoord (Galactocentric: galcen_coord=<ICRS Coordinate: (ra, dec) in deg
```

Comment:
If `CoordinateAttribute` now holds a `SkyCoord`, won't this repr change at `galcen_coord=<ICRS Coordinate:...` as well?

Reply:
`CoordinateAttribute` doesn't actually hold a `SkyCoord`.  To preserve the API, it uses a `SkyCoord`-style internal transformation, but then it drops it down to a frame with data.  (It's like the current behavior when a `SkyCoord` is provided as input.)